### ISSUE TITLE
Opaque always builds in simulation mode due to bug in CMake build

### ISSUE THREAD

**Issue by ankurdave** [ucbrise/opaque#68]
```
-if(NOT "$ENV{SGX_MODE}" EQUAL "HW")
+if(NOT "$ENV{SGX_MODE}" STREQUAL "HW")
```

Reported by András Méhes.

**Comment by ankurdave** [comment]
There is still a crash when building with `buildType := Release` on SGX hardware. Reopening.

**Comment by ankurdave** [comment]
The `buildType := Release` issue was moved to #75 and fixed in #76.
